resources/js/components/passport/AuthorizedClients.vue

```diff
@@ -0,0 +1,93 @@
+<template>
```

Comment:
Just happened to use this one:

https://github.com/codeitlikemiley/tailwind-passport-components

I know there are a few out there. This one ended up requiring me to update it for Tailwind 1.0, which was... fun.